TipCatDotNet.Api/Services/HospitalityFacilities/MemberService.cs

```diff
@@ -262,6 +263,64 @@ async Task<Result> UpdateMember()
     }
 
 
+    public Task<Result> Update(MemberContext memberContext, MemberRequest request,
+        ActiveStripeAccountType accountType, CancellationToken cancellationToken = default)
+    {
+        return ValidateGeneral(memberContext, request)
+            .Bind(DefineActiveAccount)
+            .Bind(UpdateActiveAccount);
+
+
+        async Task<Result<string>> DefineActiveAccount()
+        {
+            var activeStripeId = string.Empty;
```

Comment:
Do we allow empty strings as account IDs? If not, you may use `null` here for simplicity